_data/jobs.yml

```diff
@@ -163,10 +163,10 @@
   name: Senior Scientist Research Computing (Biomedical and Clinical Informatics)
   posted: 2021-10-27
   url: https://jobs.rutgers.edu/postings/143690
-- expires: 2021-12-30
+- expires: 2022-02-15
```

Comment:
Applications close Feb 1 2022 so maybe a bit sooner?

Reply:
Thanks for checking. I did this during a meeting and  goofed. They asked for Feb 1, I just goofed. Will fix.